tests/karma/unit/enketo/phone-widget.js

```diff
@@ -0,0 +1,116 @@
+describe('phone-widget', function() {
```

Comment:
It's encoded at https://github.com/enketo/enketo-core/blob/master/.jsbeautifyrc; basically 4 space indent, and a single space both sides of parentheses, unless they're empty in which cases there is no space inside the parens.  E.g.
```js
define( function( require, exports, module ) {
    'use strict';
    var FormModel = require( 'enketo-core/src/js/Form-model' );
    var x = function () {};
```